cdk/src/monitoring/synthetics.handler.ts

```diff
@@ -0,0 +1,49 @@
+const synthetics = require('Synthetics');
+const log = require('SyntheticsLogger');
+const syntheticsConfiguration = synthetics.getConfiguration();
+
+exports.handler = async function () {
+  // TODO: cycle through several addresses
```

Comment:
i would say priority for next tests would be to get coverage across the different types of results users might see, i.e. two more addresses, one with just water system data (no parcel) and one with neither parcel nor water system data.

Reply:
Good idea. I've made this comment more verbose to add those cases.